source/ProtonPack/ProtonPack.ino

```diff
@@ -4301,11 +4308,11 @@ void checkSerial1() {
             break;
 
             case A_SYNC_START:
-              //b_serial1_connected = false;
+              debug("Serial1 Sync Start");
```

Comment:
I would probably remove these debug calls, the strings will probably take up memory even though they are not used.